duolingo.py

```diff
@@ -120,6 +120,15 @@ def _check_login(self):
         resp = self._make_req(self.get_user_url())
         return resp.status_code == 200
 
+    def get_user_url_by_id(self, fields=None):
+    if fields is None:
+        fields = []
+        url = 'https://www.duolingo.com/2017-06-30/users/{}'.format(self.user_data.id)
+        fields_params = requests.utils.requote_uri(','.join(fields))
+        if fields_params:
+            url += '?fields={}'.format(fields_params)
+        return url
```

Comment:
This all needs un-indenting, or it'll only run when fields is None.  I guess github's suggestion editor auto-indented the rest, whoops.

Reply:
oh, didn't notice. I've fixed in one of the new commits